### DISCUSSION TITLE
Some Tag Icons are 'hidden' in Dark Mode

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#230] [Bug Reports]
For example the `Rust` tag icon is black logo, so on Dark mode it disappears into the background lol

Not sure what the best solution to this would be tho....

Dark Mode: 
![image](https://user-images.githubusercontent.com/964771/217565501-e6d784f1-d855-468c-a36b-a2350a87a830.png)

Light Mode:
![image](https://user-images.githubusercontent.com/964771/217565567-c7653d09-2de8-4895-b62e-1501ab957030.png)

**Comment by originalwebgurl** [comment]
Thanks Corey. I planned on giving these a light background and I must have overlooked applying the styles here. I just pushed a fix so they should look like this in v2.0.59 
![image](https://user-images.githubusercontent.com/2898748/217589262-ae02862d-5b2d-4173-b817-02a11a79701c.png)
![image](https://user-images.githubusercontent.com/2898748/217594693-25d34705-8f58-4c11-8030-bda3f46f10c1.png)

I'll let you know when this gets deployed so you can verify.

**Reply by coreyja** [reply]
Awesome thanks!!

I love these little icons, I noticed them for the first time this morning!

**Reply by originalwebgurl** [reply]
Agreed! Darcy did such a great job with these! 

The fix is deployed now so you should be able to see those icons in dark mode now.

**Reply by coreyja** [reply]
Looks great on my end too 🎉  Thanks for the quick fix!
